boefjes/images/base.Dockerfile

```diff
@@ -11,8 +11,8 @@ COPY $BOEFJE_PATH/requirements.txt* .
 RUN --mount=type=cache,target=/root/.cache pip install --upgrade pip && pip install httpx
 RUN if test -f requirements.txt; then pip install -r requirements.txt; fi
 
-COPY ./images/docker_adapter.py .
+COPY ./images/oci_adapter.py .
 COPY $BOEFJE_PATH $BOEFJE_PATH
 
-ENTRYPOINT ["/usr/local/bin/python", "-m", "docker_adapter"]
+ENTRYPOINT ["/usr/local/bin/python", "-m", "oci_adapter"]
```

Comment:
Just an idea: Imho `docker_adapter` (or just `adapter` or `plugin_adapter`) as a module name is more suitable, since in the near future we could also have adapters from other technologies such as kata, gVisor and Firecracker

Reply:
Yes, but it is not evident that we would need to use this same script, so for now this is a better name I would say.